linkerd/app/src/spire.rs

```diff
@@ -17,17 +17,27 @@ pub struct Config {
 
 // Connects to SPIRE workload API via Unix Domain Socket
 pub struct Client {
+    #[allow(dead_code)]
```

Comment:
can we use `cfg_attr` to only mark this type as `dead_code` for `not(target_os = "linux")`?

Reply:
makes sense, will address